controllers/suite_test.go

```diff
@@ -67,6 +67,9 @@ var _ = BeforeSuite(func(done Done) {
 	err = perfv1alpha1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
+	err = perfv1alpha1.AddToScheme(scheme.Scheme)
```

Comment:
i don't think this section is necessary

Reply:
I guess it is generated by kubebuilder. Fixed.

Reply:
I think it is generated by kubebuilder. Fixed.